input/haleu/sensitivity-analysis/global/mmr_share/mmr_share_driver.py

```diff
@@ -0,0 +1,97 @@
+import numpy as np
+import subprocess
+import dakota.interfacing as di
+import sys
+import os
+from turtle import up
+sys.path.append('../../../../../scripts')
+import create_AR_DeployInst as cdi
+import output_metrics as oup
+import dakota_input as inp
+# ----------------------------
+# Parse Dakota parameters file
+# ----------------------------
+
+params, results = di.read_parameters_file()
+
+# -------------------------------
+# Convert and send to Cyclus
+# -------------------------------
+
+# Edit Cyclus input file
+cyclus_template = 'mmr_share_input.xml.in'
+scenario_name = ('ts_' + str(int(params['ts'])) +
```

Comment:
I could be understanding the variables incorrectly, but are you worried about losing some information by turning 'mmr_share,' 'xe100_burnup,' or 'mmr_burnup' into ints (in my mind they could be floats).